src/local-ssh/ipc/extension.ts

```diff
@@ -167,92 +155,20 @@ export class ExtensionServiceServer extends Disposable {
     }
 
     private async tryActive() {
-        this.logService.info('going to start extension ipc service server with id', this.id);
-        this.server.listen('127.0.0.1:0').then((port) => {
-            this.ipcPort = port;
-            this.logService.info('extension ipc service server started to listen with id: ' + this.id);
-            this.pingLocalSSHService();
-            this.backoffActive();
+        const port = Configuration.getLocalSshExtensionIpcPort();
+        this.logService.debug('going to try active extension ipc service server on port ' + port);
+        this.server.listen('127.0.0.1:' + port).then(() => {
+            this.logService.info('extension ipc service server started to listen');
         }).catch(e => {
-            this.logService.error(e, `extension ipc service server failed to listen with id: ${this.id}`);
+            this.logService.debug(`extension ipc service server failed to listen`, e);
+            // TODO(lssh): listen to port and wait until disconnect and try again
+            timeout(1000).then(() => {
+                this.tryActive();
+            });
```

Comment:
Maybe we need to connect to ensure the port is used by ourselves and not another app, if it's another app then show a message to user